go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/minio/dperf v0.3.4
 	github.com/minio/highwayhash v1.0.2
 	github.com/minio/kes v0.19.0
-	github.com/minio/madmin-go v1.3.5
+	github.com/minio/madmin-go v1.3.9
```

Comment:
please move to v1.3.11 and also update the `minio/mc` dependency.  Also change `go 1.16` in  in `go.mod` to `go 1.17` and run `go mod tidy -compat=1.17`

Reply:
Done